presto-native-execution/presto_cpp/main/types/tests/RowExpressionTest.cpp

```diff
@@ -490,6 +538,80 @@ TEST_F(RowExpressionTest, call) {
   }
 }
 
+TEST_F(RowExpressionTest, castToVarchar) {
+  // CAST(varchar_col AS varchar)
+  {
+    json j = json::parse(makeCastToVarchar(false, "varchar", "varchar"));
+    std::shared_ptr<protocol::CallExpression> p = j;
```

Comment:
Can we assign p directly without having variable j in between ?

Reply:
Yes, it works. Revising.

(Thought about it before, but turned out using the same pattern as the rest of unit tests.)